pulp_ansible/app/models.py

```diff
@@ -499,6 +907,19 @@ class AnsibleRepository(Repository, AutoAddObjPermsMixin):
     gpgkey = models.TextField(null=True)
     private = models.BooleanField(default=False)
 
+    sigstore_signing_service = models.ForeignKey(
+        SigstoreSigningService,
+        on_delete=models.SET_NULL,
+        related_name="ansible_repositories",
+        null=True,
+    )
```

Comment:
```suggestion
```
Unless you merge the two models together, the repository should only have a reference to the verifier.